cmd/delete/delete_test.go

```diff
@@ -32,16 +32,16 @@ func TestDeleteCommand(t *testing.T) {
 	assert.Equal(t, deleteCommand.Use, "delete")
 	assert.Equal(t, deleteCommand.Short, deleteCmdShort)
 	assert.Equal(t, deleteCommand.Long, deleteCmdLong)
-	assert.Equal(t, len(deleteCommand.Commands()), 5)
+	assert.Equal(t, len(deleteCommand.Commands()), 6)
 	cmdNouns := deleteCommand.Commands()
 	// Sort by Use value.
 	sort.Slice(cmdNouns, func(i, j int) bool {
 		return cmdNouns[i].Use < cmdNouns[j].Use
 	})
-	useArray := []string{"cluster-resource-attribute", "execution", "execution-cluster-label", "execution-queue-attribute", "task-resource-attribute"}
-	aliases := [][]string{{"cluster-resource-attributes"}, {"executions"}, {"execution-cluster-labels"}, {"execution-queue-attributes"}, {"task-resource-attributes"}}
-	shortArray := []string{clusterResourceAttributesShort, execCmdShort, executionClusterLabelShort, executionQueueAttributesShort, taskResourceAttributesShort}
-	longArray := []string{clusterResourceAttributesLong, execCmdLong, executionClusterLabelLong, executionQueueAttributesLong, taskResourceAttributesLong}
+	useArray := []string{"cluster-resource-attribute", "execution", "execution-cluster-label", "execution-queue-attribute", "plugin-override", "task-resource-attribute"}
```

Comment:
why do these change?

Reply:
These are required for the tests whenever new noun is added as a command.  It just adding a new noun to the array which got supported in this PR.